extensions/parquet/base/src/main/java/io/deephaven/parquet/base/materializers/LocalDateTimeFromNanosMaterializer.java

```diff
@@ -0,0 +1,53 @@
+//
+// Copyright (c) 2016-2024 Deephaven Data Labs and Patent Pending
+//
+// ****** AUTO-GENERATED CLASS - DO NOT EDIT MANUALLY
+// ****** Edit LocalDateTimeFromMicrosMaterializer and run "./gradlew replicatePageMaterializers" to regenerate
```

Comment:
I'm curious why we picked micros as the standard, rather than millis or nanos (which I would have expected).

Reply:
Its a silly thing. 

Basically, in case of `LocalTime`, I went from Micros-> Millis + Nanos because Micros used `long`, Millis used `int`, and Nanos used `long`. So it was easier to convert from `long` to `int` than `int` to `long`, because there are some other `int` variables in these classes.

So I kept the same pattern in `LocalDateTime` as well, going from Micros-> Millis + Nanos.
If you look at `ReplicatePageMaterializers`, the pattern seems more consistent. But I can very well change `LocalDateTime` to go Millis-> Micros + Nanos, if it makes more intuitive sense.